src/Spec.ts

```diff
@@ -1011,7 +1011,8 @@ ${await utils.readFile(path.join(__dirname, '../js/multipage.js'))}
       const baseIndentRe = new RegExp('^' + baseIndent, 'gm');
       input = input.replace(baseIndentRe, '');
 
-      const result = hljs.highlight(lang, input);
+      // @ts-expect-error the type definitions for highlight.js are broken
```

Comment:
The types say that there's a default export with a bunch of properties (including `highlight`). In reality, there is no default export, only named exports. TypeScript doesn't allow you to pull named exports it doesn't know about, and using the default export gets you a runtime error that `highlight` is not a property of `undefined`. The types are broken, so we ignore them.